src/os_auth/local-server.c

```diff
@@ -405,7 +405,7 @@ cJSON* local_remove(const char *id, int purge) {
     w_mutex_lock(&mutex_keys);
 
     if (index = OS_IsAllowedID(&keys, id), index < 0) {
-        merror("ERROR %d: %s.", ERRORS[ENOAGENT].code, ERRORS[ENOAGENT].message);
+        mdebug1("Error %d: %s.", ERRORS[ENOAGENT].code, ERRORS[ENOAGENT].message);
```

Comment:
Line 433 of local-server.c has the same log for the **get** option. I think that log should be a debug too.
And both messages can be identical.

Reply:
Updated!